src/celldega/pre/trx_tile.py

```diff
@@ -163,7 +174,7 @@ def make_trx_tiles(
         os.makedirs(path_trx_tiles)
 
     transformation_matrix = np.loadtxt(path_transformation_matrix)
-
+    
```

Comment:
lets revert any changes that add whitespace. You can adjust your text editor to to this by default. We might also start using a code formatter like Python black.